sample/src/test/java/app/cash/paparazzi/sample/VisibilityA11yTest.kt

```diff
@@ -0,0 +1,56 @@
+package app.cash.paparazzi.sample
+
+import android.content.Context
+import android.view.View
+import android.view.View.GONE
+import android.view.View.INVISIBLE
+import android.view.ViewGroup
+import android.view.ViewGroup.LayoutParams.WRAP_CONTENT
+import android.widget.LinearLayout
+import android.widget.TextView
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.accessibility.AccessibilityRenderExtension
+import org.junit.Rule
+import org.junit.Test
+
+class VisibilityA11yTest {
```

Comment:
let's remove this test, since the change to MixedView should be covered by an already written test.

Reply:
Sounds good!